stdlib/universe/filter.go

```diff
@@ -146,7 +146,10 @@ func (t *filterTransformation) Process(id execute.DatasetID, tbl flux.Table) err
 	// Prepare the function for the column types.
 	cols := tbl.Cols()
 	if err := t.fn.Prepare(cols); err != nil {
-		// TODO(nathanielc): Should we not fail the query for failed compilation?
+		if _, ok := err.(*execute.UnknownColumnError); ok {
```

Comment:
Referencing an unknown column in other contexts is an error. Why in this case do we ignore it and just assume the predicate evals to false?

Reply:
hm.  maybe I ought to pull this into another PR.  and get more discussion going on it.  

You are right, this is wrong, not in the sense that there should sometimes be an error.  In the context of filters, I don't think a missing column should ever be an error.  I'll make a new issue with more details and remove this change from this PR.